dom.bs

```diff
@@ -675,6 +680,12 @@ The <dfn attribute for=Event><code>bubbles</code></dfn> and
 <dfn attribute for=Event><code>cancelable</code></dfn> attributes
 must return the values they were initialized to.
 
+<p>The <dfn attribute for=Event><code>returnValue</code></dfn> attribute must be initialized to true
+when an <a>event</a> is created. When the attribute is set to false it must set the <a>canceled flag</a>
```

Comment:
> When the attribute is set to

Should be phrased as a setter instead?: "The `returnValue` attribute’s setter must [...]"

[See `cancelBubble`](https://dom.spec.whatwg.org/#dom-event-cancelbubble) for an example of the getter/setter boilerplate verbiage.

Reply:
Yeah, I think phrasing this as a getter and setter (one paragraph each) (and not saying anything about initial state) would be best. The initial state doesn't really exist, since it just reflects the canceled flag.